lib/bots/syncbot.ts

```diff
@@ -373,7 +372,7 @@ export class SyncBot extends ProcBot {
                 event.toIds[type] = this.rooms[from][id][to];
                 resolve(this.rooms[from][id][to]);
             } else {
```

Comment:
@hedss Correct me if I'm wrong, but without the else it'll resolve and also error since resolve isn't return and doesn't stop execution.